static/js/HitPoint.js

```diff
@@ -1,13 +1,25 @@
+var socket = io.connect(window.location.protocol + '//' + document.domain + ':' + location.port);
+var i=0;
+socket.on('out_pipe', function () {
+	console.log("out_pipe", socket.connected)
+    const heartList=['heart0','heart1','heart2','heart3','heart4'];
+    heartBroken(heartList[i]);
+    changeimg(heartList[i]);
+});
+
```

Comment:
Rewrite as below:
Receive heart numbe  from websocket, and the function prototype should be `function(data) { ... }`,
 read the heart number from data and remove the global variable `i`.

Try to imagine, if we want to change the heart number, is this code good?